core/Archive/ArchiveInvalidator.php

```diff
@@ -310,7 +310,7 @@ public function markArchivesAsInvalidated(array $idSites, array $dates, $period,
          *
          * @param array &$idSites An array containing a list of site IDs which are requested to be invalidated.
          */
-        Piwik::postEvent('Archiving.getIdSitesToMarkArchivesAsInvalidated', array(&$idSites));
+        Piwik::postEvent('Archiving.getIdSitesToMarkArchivesAsInvalidated', array(&$idSites, $segment));
```

Comment:
Passing only the segment here actually looks a bit random. Why not also passing the dates, period or the name. Might all be something that could be relevant in some cases.
Also when updating events, you should update the comment above, so it describes all parameters. And it might be good to mention the change in the changelog, so developers are aware of it.

Reply:
@sgiehl Started passing `dates`, `period`, `segment` and `name` parameter as suggested.
Also updated `CHANGELOG.md` please check if that't the correct way to update it.